ReturnReminderManager/mail_client.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 21 20:23:41 2021
+
+@author: Simon
+"""
+import os
+import base64
+import logging
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+logging.basicConfig(level=logging.INFO)
+
+
+class MailClient:
+
+    def __init__(self):
+        self._authenticate()
+
+    def _authenticate(self):
+        user = os.environ.get("MAIL_USER")
+        password = os.environ.get("MAIL_PASSWORD")
+        server = os.environ.get("MAIL_SERVER")
+        logging.debug(f'logging in to SMTP: {server} with mail {user}')
+        server = smtplib.SMTP(server, timeout=15)
+        server.starttls()
+        server.login(user, password)
+        self.server = server
+        self.leihlokal_mail = base64.b64decode(b'bGVpaGxva2FsQGJ1ZXJnZXJzdGlmdHVuZy1rYXJsc3J1aGUuZGU=').decode()
```

Comment:
What is this for? If you don‘t want the mail to be visible you could add it as a secret.

Reply:
I just copied it from https://github.com/leih-lokal/scripts/blob/master/WPAppointmentManager/mail_client.py

The mail is not secret, but this way we protect ourselves a bit from spam I guess